HarpGuru/HarpFace/index.ts

```diff
@@ -1,5 +1,4 @@
-export { HarpFace } from './components'
-export { DisplayModes } from './types'
-export type { HarpFaceProps } from './types'
+export { HarpFace, DisplayModes } from './HarpFace'
```

Comment:
Should displaymodes be a type definition at a higher level since it's used in the control panel as well as the harpface.

Reply:
displayMode changed to activeDisplayMode where relevant. This includes renaming setDisplayMode to setActiveDisplayMode.